src/Asm89/Stack/Cors.php

```diff
@@ -53,10 +53,6 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
             return $this->cors->handlePreflightRequest($request);
         }
 
-        if (!$this->cors->isActualRequestAllowed($request)) {
```

Comment:
I would also remove `CorsService::isActualRequestAllowed()` since it doesn't appear to be used now.

Reply:
Yes but removing public methods is really really a breaking change :)

Reply:
I see, might be wise to throw a deprecation warning then. :)